src/api.ts

```diff
@@ -26,7 +26,7 @@ router.post('/scores', async (req, res) => {
   const { space = '', network = '1', snapshot = 'latest', addresses = [] } = params;
   let { strategies = [] } = params;
   // strategy parameters should be same order to maintain consistent key hashes
-  strategies = Array.isArray(strategies) ? strategies.map(sortObjectByParam) : [];
+  strategies = Array.isArray(strategies) ? strategies.map(updateStrategyNetwork(network)).map(sortObjectByParam) : [];
```

Comment:
Can you make a function formatStrategies that take all logic. You can do strategies = formatStrategies(strategies, network);